change/@microsoft-teams-js-286cc515-1cef-4d54-9af6-d869e00d96c9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
I think changing the function signature would be a breaking change for MChat.

Reply:
Because it's marked beta, we have some leeway. We will work with them to ensure they're expecting this.

Reply:
oh, since the API was marked @beta, it would still be a minor change :)

Reply:
haha it looks like we commented at the same time